src/aura/BGE_DataImportBatchEntry/BGE_DataImportBatchEntryHelper.js

```diff
@@ -9,7 +9,6 @@
             if (state === 'SUCCESS') {
```

Comment:
@ptewson-sfdo this is the file that I'd like to reformat to match the helpful grouping we did with the Config Wizard. If you see anything else that needs reformatting/reordering, feel free to call it out!